src/neo/neo.csproj

```diff
@@ -21,7 +21,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Akka" Version="1.3.16" />
+    <PackageReference Include="Akka" Version="1.4.1-rc1" />
```

Comment:
suggest that we should use a release version. this seems to be a release candidate one.

Reply:
@cloud8little Yes, There is no release version of akka 1.4. We can wait for the release version. I will test the new akka first.